app/pages/admin/project-status-list.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import apiClient from 'panoptes-client/lib/api-client';
+import LoadingIndicator from '../../components/loading-indicator';
+import ProjectIcon from '../../components/project-icon';
+import Paginator from '../../talk/lib/paginator';
+
+class ProjectStatusList extends Component {
+  constructor(props) {
+    super(props);
+    this.getProjects = this.getProjects.bind(this);
+    this.renderProjectList = this.renderProjectList.bind(this);
+    this.renderProjectListItem = this.renderProjectListItem.bind(this);
+    this.state = {
+      loading: false,
+      projects: [],
+    };
+  }
+
+  componentWillMount() {
+    this.getProjects();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.location.query !== this.props.location.query) {
+      this.getProjects();
+    }
+  }
+
+  getProjects() {
+    const { query } = this.props.location;
+    const projectsQuery = {
+      include: 'avatar',
+      page_size: 24,
+      sort: '-updated_at',
+    };
+
+    if (query && query.filterBy && !query.slug) {
+      projectsQuery[query.filterBy] = true;
+    }
+
+    this.setState({ loading: true });
+    return apiClient.type('projects').get(projectsQuery)
+      .then((projects) => this.setState({ projects, loading: false }))
+      .catch((error) => console.error('Error requesting projects', error));
+  }
+
+  renderProjectList() {
+    const { projects } = this.state;
+    const meta = projects[0].getMeta() || {};
+    return (projects.length === 0)
```

Comment:
I know it's just a style preference, but I find this kind of multi-line ternary JSX formatting *really* hard to read.